updated_main.py

```diff
@@ -3,57 +3,89 @@
 
     soup = bs4.BeautifulSoup(request_result.text, "html.parser")
-
     temp = soup.find("div", class_="BNeawe iBp4i AP7Wnd").text
-
     temps = temp.split(" ")
     current_price = float(temps[0])
-
-    # Using webhook we send the message in the Discord server
+    
+    
+def discord_bot():
+    """Using webhook we send the message in the Discord server"""
     hook = Webhook(
-        "https://discord.com/api/webhooks/1003993992360366080/1cS8w3N2GHzW-2aAwLYZ9tpfcHHhRBiASgTIafnfJ1Q--NbC1Exat7lVVzr4mgvH3l5f"
-    )
+            "https://discord.com/api/webhooks/1003993992360366080/1cS8w3N2GHzW-2aAwLYZ9tpfcHHhRBiASgTIafnfJ1Q--NbC1Exat7lVVzr4mgvH3l5f"
+        )
 
     # Defining the upper and lower limit to get the alert when it will be triggered
     upper_limit = 96.50
     lower_limit = 96.40
-
+    
     if current_price >= upper_limit:
         high_price_alert = (
-            "ALERT HIGH PRICE: Current price of 1 British Pound is = "
-            + str(current_price)
-            + " Indian Rupees"
-        )
+                "ALERT HIGH PRICE: Current price of 1 British Pound is = "
+                + str(current_price)
+                + " Indian Rupees"
+            )
         hook.send(high_price_alert)
         upper_limit += 0.50
 
     elif current_price <= lower_limit:
         low_price_alert = (
-            "ALERT LOW PRICE: Current price of 1 British Pound is = "
-            + str(current_price)
-            + " Indian Rupees"
-        )
+                "ALERT LOW PRICE: Current price of 1 British Pound is = "
+                + str(current_price)
+                + " Indian Rupees"
+            )
         hook.send(low_price_alert)
         lower_limit -= 0.50
 
     else:
         current_price_alert = (
-            "Current price of 1 British Pound is = "
-            + str(current_price)
-            + " Indian Rupees"
-        )
+                "Current price of 1 British Pound is = "
+                + str(current_price)
+                + " Indian Rupees"
+            )
         hook.send(current_price_alert)
+            
+            
+def send_email():
+    """#Sneding automated email"""
+    sender = 'mohitpatni0731@gmail.com'
+    password = 'ckyryrdajffgmnde'
+    reciever = 'mohitpatni786@gmail.com'
+
+    subject = 'AUTOMATED Pound rate email'
+    body = 'Current price of 1 British Pound is = '+str(current_price)+' Indian Rupees'
+
+    em = EmailMessage()
+    em['From'] = sender
+    em['To'] = reciever
+    em['Subject'] = subject
+    em.set_content(body)
+
+    context = ssl.create_default_context()
+
+    with smtplib.SMTP_SSL('smtp.gmail.com', 465, context=context) as smtp:   
+        smtp.login(sender, password)
+        smtp.sendmail(sender, reciever, em.as_string())
+  
+
+# Running a for loop to get the latest price of the GBP
+While True:
+    getting_current_price()
+    discord_bot()
+    send_email()
+    
+time.sleep(1000)
```

Comment:
Why should we sleep here?

Reply:
To run the script after some time for example after half an hour. Alternatively, we can use any time modules to loop after a particular number of minutes.

Reply:
Use `cron` for that https://linuxhint.com/cron_jobs_complete_beginners_tutorial/.

Cron is a linux command which lets you schedule run of scripts at particular times.

Reply:
But if use cron then user has to run his code in linux only. Also there is task scheduler in windows also but that also limits the user to run only on the windows not on any cloud like google collab etc.

So what we can do is by using schedule and time module in python we can run the script after particular time

Reply:
`schedule.every(1).minutes.do(getting_current_price, discord_bot, send_email)
while 1:
    schedule.run_pending()
    time.sleep(1)`
like this